eng/restore/harvestPackages.targets

```diff
@@ -1,15 +1,12 @@
-﻿<Project InitialTargets="AddPackageDownload" Sdk="Microsoft.NET.Sdk">
+﻿<Project InitialTargets="AddPackageDownload">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <PackagingTaskAssembly>$(NuGetPackageRoot)microsoft.dotnet.build.tasks.packaging\$(MicrosoftDotNetBuildTasksPackagingVersion)\tools\</PackagingTaskAssembly>
```

Comment:
Can we instead use the Package property that NuGet  defined in the generated props? This is restored in tools.proj and we import those generated targets, right?

Reply:
Unfortunately not as this target adds to the restore phase in which the generated properties aren't available.

Reply:
OK I was missing one additional detail. The package is restored in Tools.props means in Arcade's Tools.proj so we can't use it in the restore phase of packages.proj where this is imported.